ADOPTERS.md

```diff
@@ -0,0 +1,7 @@
+# Who uses GitOps? 
+
+As the GitOps movement grows, we'd like to hear from organizations who are successfully implementing GitOps! Please send a PR with *your* organization name if you are adopting the [GitOps principles](https://opengitops.dev/#principles) in practice. Please include how you are adopting.
```

Comment:
I'd change movement to adoption

GitOps isn't a movement in so much that it's a move to a better way to do things.

Reply:
I agree with @chris-short, adoption is a better fit in this case.

Reply:
Yeah fair point, we can remove that. I do like GitOps as a movement but we don't have to plug it here.